src/Components/CreateImageWizard/steps/Kernel/components/KernelArguments.tsx

```diff
@@ -1,9 +1,133 @@
-import React from 'react';
+import React, { useState } from 'react';
 
-import { FormGroup } from '@patternfly/react-core';
+import {
+  Button,
+  Chip,
+  ChipGroup,
+  FormGroup,
+  HelperText,
+  HelperTextItem,
+  TextInputGroup,
+  TextInputGroupMain,
+  TextInputGroupUtilities,
+} from '@patternfly/react-core';
+import { PlusCircleIcon, TimesIcon } from '@patternfly/react-icons';
+
+import { useAppDispatch, useAppSelector } from '../../../../../store/hooks';
+import {
+  addKernelArg,
+  removeKernelArg,
+  selectKernel,
+} from '../../../../../store/wizardSlice';
+import { isKernelArgumentValid } from '../../../validators';
 
 const KernelArguments = () => {
-  return <FormGroup isRequired={false} label="Append"></FormGroup>;
+  const dispatch = useAppDispatch();
+  const kernelAppend = useAppSelector(selectKernel).append;
+
+  const [inputValue, setInputValue] = useState('');
+  const [errorText, setErrorText] = useState('');
+
+  const onTextInputChange = (
+    _event: React.FormEvent<HTMLInputElement>,
+    value: string
+  ) => {
+    setInputValue(value);
+  };
+
+  const handleKeyDown = (e: React.KeyboardEvent, value: string) => {
+    if (e.key === 'Enter') {
+      e.preventDefault();
+
+      if (
+        isKernelArgumentValid(value) &&
+        !kernelAppend.some((arg) => arg.name === value)
+      ) {
+        dispatch(addKernelArg({ name: value, isRequiredByOpenSCAP: false }));
+        setInputValue('');
+        setErrorText('');
+      }
+
+      if (kernelAppend.some((arg) => arg.name === value)) {
```

Comment:
should I see this error in case I am adding the same kernel name?
now I dont see any error when adding the same name

Reply:
Tried to add a duplicate and got the correct error
![image](https://github.com/user-attachments/assets/18662507-b8e2-43ba-ada9-4d28a57ace51)

What arguments did you try to add?

Reply:
I tried 'true' value

Reply:
ok, I got you, I think I check rhat I am adding 2 time the same value 'test',
if I tried to add same value that comes from the oscap profile the validation is working. 
but, maybe we also want to check if user try to add same value ?